[2022-07-04 15:57:03,971: INFO: stage_01_prepare_data]: 
********************
[2022-07-04 15:57:03,971: INFO: stage_01_prepare_data]: >>>>> stage Prepare_Data started <<<<<
[2022-07-04 15:57:03,971: ERROR: stage_01_prepare_data]: [Errno 2] No such file or directory: 'configs/config.yaml'
Traceback (most recent call last):
  File "src/stage_01_prepare_data.py", line 51, in <module>
    main(config_path=parsed_args.config, params_path=parsed_args.params)
  File "src/stage_01_prepare_data.py", line 22, in main
    config = read_yaml(config_path)
  File "d:\myeuron\fulstackdatascientist\dvc-nlp-doc-project\dvc-nlp-project-with-docs\src\utils\common.py", line 9, in read_yaml
    with open(path_to_yaml) as yaml_file:
FileNotFoundError: [Errno 2] No such file or directory: 'configs/config.yaml'
[2022-07-04 15:57:44,130: INFO: stage_01_prepare_data]: 
********************
[2022-07-04 15:57:44,130: INFO: stage_01_prepare_data]: >>>>> stage Prepare_Data started <<<<<
[2022-07-04 15:57:44,133: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-07-04 15:57:44,134: INFO: common]: yaml file: params.yaml loaded successfully
[2022-07-04 15:57:44,134: ERROR: stage_01_prepare_data]: 'artificats'
Traceback (most recent call last):
  File "src/stage_01_prepare_data.py", line 51, in <module>
    main(config_path=parsed_args.config, params_path=parsed_args.params)
  File "src/stage_01_prepare_data.py", line 34, in main
    artificats = config["artificats"]
KeyError: 'artificats'
[2022-07-04 16:24:42,280: INFO: stage_01_prepare_data]: 
********************
[2022-07-04 16:24:42,280: INFO: stage_01_prepare_data]: >>>>> stage Prepare_Data started <<<<<
[2022-07-04 16:24:42,283: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-07-04 16:24:42,284: INFO: common]: yaml file: params.yaml loaded successfully
[2022-07-04 16:24:42,284: ERROR: stage_01_prepare_data]: name 'artifacts' is not defined
Traceback (most recent call last):
  File "src/stage_01_prepare_data.py", line 51, in <module>
    main(config_path=parsed_args.config, params_path=parsed_args.params)
  File "src/stage_01_prepare_data.py", line 35, in main
    prepare_data_dir_path = os.path.join(artifacts["ARTIFACTS_DIR"], artifacts["PREPARED_DATA"])
NameError: name 'artifacts' is not defined
[2022-07-04 16:28:14,673: INFO: stage_01_prepare_data]: 
********************
[2022-07-04 16:28:14,673: INFO: stage_01_prepare_data]: >>>>> stage Prepare_Data started <<<<<
[2022-07-04 16:28:14,675: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-07-04 16:28:14,676: INFO: common]: yaml file: params.yaml loaded successfully
[2022-07-04 16:28:14,676: ERROR: stage_01_prepare_data]: name 'artifacts' is not defined
Traceback (most recent call last):
  File "src/stage_01_prepare_data.py", line 51, in <module>
    main(config_path=parsed_args.config, params_path=parsed_args.params)
  File "src/stage_01_prepare_data.py", line 35, in main
    prepare_data_dir_path = os.path.join(artifacts["ARTIFACTS_DIR"], artifacts["PREPARED_DATA"])
NameError: name 'artifacts' is not defined
[2022-07-04 16:30:33,257: INFO: stage_01_prepare_data]: 
********************
[2022-07-04 16:30:33,257: INFO: stage_01_prepare_data]: >>>>> stage Prepare_Data started <<<<<
[2022-07-04 16:30:33,260: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-07-04 16:30:33,260: INFO: common]: yaml file: params.yaml loaded successfully
[2022-07-04 16:30:33,261: INFO: common]: created directory at: artifacts\prepared
[2022-07-04 16:30:33,261: INFO: stage_01_prepare_data]: >>>>> stage Prepare_Data completed!<<<<<

[2022-07-04 16:32:35,817: INFO: stage_01_prepare_data]: 
********************
[2022-07-04 16:32:35,817: INFO: stage_01_prepare_data]: >>>>> stage Prepare_Data started <<<<<
[2022-07-04 16:32:35,819: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-07-04 16:32:35,820: INFO: common]: yaml file: params.yaml loaded successfully
[2022-07-04 16:32:35,820: ERROR: stage_01_prepare_data]: name 'artifacts' is not defined
Traceback (most recent call last):
  File "src/stage_01_prepare_data.py", line 51, in <module>
    main(config_path=parsed_args.config, params_path=parsed_args.params)
  File "src/stage_01_prepare_data.py", line 35, in main
    prepare_data_dir_path = os.path.join(artifacts["ARTIFACTS_DIR"], artifacts["PREPARED_DATA"])
NameError: name 'artifacts' is not defined
[2022-07-04 16:33:44,265: INFO: stage_01_prepare_data]: 
********************
[2022-07-04 16:33:44,265: INFO: stage_01_prepare_data]: >>>>> stage Prepare_Data started <<<<<
[2022-07-04 16:33:44,268: INFO: common]: yaml file: configs/config.yaml loaded successfully
[2022-07-04 16:33:44,269: INFO: common]: yaml file: params.yaml loaded successfully
[2022-07-04 16:33:44,269: INFO: common]: created directory at: artifacts\prepared
[2022-07-04 16:33:44,269: INFO: stage_01_prepare_data]: >>>>> stage Prepare_Data completed!<<<<<

